### PR TITLE
story/VOGRE-19

### PR DESCRIPTION
**Pull request by rajvi-patel-22**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
https://diging.atlassian.net/browse/VOGRE-19

## Anything else the reviewer needs to know?
- Fixed date appellation sidebar
    - Issue: Date Appellations were showing both date and text appellations
- Fixed select button for relation
    - Issue: Relation list item was not getting highlighted when we select it

### CODE REVIEW

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

File: annotations/static/annotations/js/annotators/appellationlist.js [hunk 1]
```diff
@@ -217,6 +217,7 @@ AppellationList = {
             // Replace an array prop wholesale doesn't seem to trigger a
             //  DOM update in the v-for binding, but a push() does; so we'll
             //  just push the appellations that aren't already in the array.
+            this.current_appellations = [];
```
Comment (jdamerow) [inline annotations/static/annotations/js/annotators/appellationlist.js]: I'm confused, wouldn't this make the next line useless? Doesn't the next line create a list of ids of current appellations, which now would always be an empty list?

Comment (rajvi-patel-22) [reply annotations/static/annotations/js/annotators/appellationlist.js]: That's right. we don't need those lines.  Just `this.current_appellations = this.appellations;` will work. 
Can you confirm if the functionality of the buttons mentioned below is correct?
Screenshot 2024-12-06 at 10 06 22 AM

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Pull request merged**